File: C++/1060 - Números Positivos.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
	float n;
	int i=0,cont=0;
	do{
		cin>>n;
		if(n>0){
			cont=cont+1;
		}
		i++;
	}while(i<6);
	cout<<cont<<" valores positivos\n";
 
    return 0;
}

File: Java/1060 - Números Positivos.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
  Scanner input = new Scanner(System.in);
        float n;
	int i=0,cont=0;
        String temp;
	do{
		n = input.nextFloat();
		if(n>0){
		cont = cont+1;
		}
		i++;
	}while(i<6);
	System.out.print(cont+" valores positivos\n");
        }  
    }